tutorial/chapter1.md

```diff
@@ -314,6 +310,27 @@ int main(int argc, char *argv[])
     }
 }
 ```
+Once, a quit sequence is pressed by the user join() will be called for our `imageProcessingServiceThread` and sequentially we will shutdown `fileImageStreamingService`, `imageProcessingServices`, we will remove our listeners, and, stop the event loop. 
+
+At this point, it is considered useful to understand the code of the [image_processor_service.cpp](../modules/image_processing/src/image_processor_service.cpp)  / [image_processor_service.h](../modules/image_processing/include/image_processing/image_processor_service.h) 
```

Comment:
I think a brief explanation on those classes would improve the readme. As a suggestion, check if you can reference concepts explained in kpsr/kpe-tutorial with direct links.

If you consider it, emphasize that the `if (started)` in the `onImageDataReceived` callback is needed only by DataMux provider and explain why.